lldb/packages/Python/lldbsuite/test/tools/lldb-dap/dap_server.py

```diff
@@ -955,6 +955,13 @@ def request_setBreakpoints(self, file_path, line_array, data=None):
         """
         (dir, base) = os.path.split(file_path)
         source_dict = {"name": base, "path": file_path}
+        return self.request_setBreakpoints_with_source(source_dict, line_array, data)
+
+    def request_setBreakpointsAssembly(self, sourceReference, line_array, data=None):
+        source_dict = {"sourceReference": sourceReference}
+        return self.request_setBreakpoints_with_source(source_dict, line_array, data)
+
+    def request_setBreakpoints_with_source(self, source_dict, line_array, data=None):
```

Comment:
Could we keep the `request_*` names just the DAP requests and make a helper without the `request_*` prefix for this? I like the consistency of having `request_*` being able to map to specific request and I think its valuable to keep that.